debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/SqlUtils.java

```diff
@@ -145,35 +145,38 @@ static String oldestFirstChangeQuery(Duration archiveLogRetention) {
         return sb.append(")").toString();
     }
 
-    public static String allOnlineLogsQuery() {
-        return String.format("SELECT MIN(F.MEMBER) AS FILE_NAME, L.NEXT_CHANGE# AS NEXT_CHANGE, F.GROUP#, L.FIRST_CHANGE# AS FIRST_CHANGE, L.STATUS " +
-                " FROM %s L, %s F " +
-                " WHERE F.GROUP# = L.GROUP# AND L.NEXT_CHANGE# > 0 " +
-                " GROUP BY F.GROUP#, L.NEXT_CHANGE#, L.FIRST_CHANGE#, L.STATUS ORDER BY 3", LOG_VIEW, LOGFILE_VIEW);
-    }
-
     /**
-     * Obtain the query to be used to fetch archive logs.
+     * Obtain a query to fetch all available minable logs, both archive and online redo logs.
      *
-     * @param scn oldest scn to search for
+     * @param scn oldest system change number to search by
      * @param archiveLogRetention duration archive logs will be mined
-     * @return query
+     * @return the query string to obtain minable log files
      */
-    public static String archiveLogsQuery(Scn scn, Duration archiveLogRetention) {
+    public static String allMinableLogsQuery(Scn scn, Duration archiveLogRetention) {
         final StringBuilder sb = new StringBuilder();
-        sb.append("SELECT NAME AS FILE_NAME, NEXT_CHANGE# AS NEXT_CHANGE, FIRST_CHANGE# AS FIRST_CHANGE ");
-        sb.append("FROM ").append(ARCHIVED_LOG_VIEW).append(" ");
-        sb.append("WHERE NAME IS NOT NULL ");
-        sb.append("AND ARCHIVED = 'YES' ");
-        sb.append("AND STATUS = 'A' ");
-        sb.append("AND NEXT_CHANGE# > ").append(scn).append(" ");
-        sb.append("AND DEST_ID IN (").append(localArchiveLogDestinationsOnlyQuery()).append(") ");
+        sb.append("SELECT MIN(F.MEMBER) AS FILE_NAME, L.FIRST_CHANGE# FIRST_CHANGE, L.NEXT_CHANGE# NEXT_CHANGE, L.ARCHIVED, ");
```

Comment:
I think it is time to start documenting the queries. COuld you please add a comment with the final query and describe the fragments explaining what each does? Thnaks

Reply:
Pushed a commit with the requested changes.